Assets/js/resultspage.js

```diff
@@ -0,0 +1,169 @@
+    async: true,
+    dataType: "json",
+    success: function (response) {
+      // Error handling if no events are returned.
+      if (response?._embedded?.events?.length === undefined) {
+        $("#eventError").text(
+          "Sorry, there are no upcoming events in this area."
+        );
+        $("#eventErrorSig").text("Sincerely, COVID-19");
+      }
+      // If events are found, run the following
+      else {
+        // Get a random number between 0 and legnth of array
+        let randomEventIndex = Math.floor(
+          Math.random() * response._embedded.events.length
+        );
+        let eventData = response._embedded.events[randomEventIndex];
+
+        // Creating <img> element and adding image url to src attr.
+        let eventImage = $("<img>").attr({
+          src: eventData?.images[0].url,
+          height: "auto",
+          width: "100%",
+          fit: "scale",
+        });
+        eventImage.remove();
+
+        // Event image
+        if (eventData?.images[0].url !== undefined || "") {
+          $("#eventImage").empty();
+          $("#eventImage").append(eventImage);
+        }
+        // Event name
+        $("#eventName").text(`Name: ${eventData.name}`);
+        // Convert date format to MM-DD-YYYY using moment.js
+        if (eventData.dates.start.localDate !== undefined || "") {
+          $("#eventDate").text(
+            `Date: ${moment(eventData?.dates?.start?.localDate).format("L")}`
+          );
+        }
+        // Event description
+        if (eventData.info !== undefined || "") {
+          $("#eventInfo").text(`Description: ${eventData?.info}`);
+        }
+        // Link to event
+        if (eventData.url !== undefined || "") {
+          // Creating anchor element to hold ticket link
+          let ticketLink = $("<a>")
+            .text("Get Tickets")
+            .attr("href", eventData.url);
+          $("#eventTickets").text(`Tickets: `);
+          $("#eventTickets").append(ticketLink);
+        }
+        let newEventButton = $("<button>")
+          .addClass("newEventBtn btn waves-effect waves-light")
+          .text("New Event Option");
+        $("#eventBtnHolder").html(newEventButton);
+        $(".newEventBtn").on("click", function () {
+          newEventButton.prop("disabled", true);;
+          eventRequest(zipCode);
+        });
+      }
+    },
+    error: function (xhr, status, err) {
+      console.log("ERROR == ", xhr.status);
+    },
+  });
+}
+
+$("#emailBtn").on("click", function () {
+  localStorage.clear();
+  let choicesArr = JSON.parse(localStorage.getItem("choices")) || [];
+  choicesArr.push({
+    food: $("#restName").text(),
+    event: $("#eventName").text(),
+  });
+  localStorage.setItem("choices", JSON.stringify(choicesArr));
+  choicesArr = JSON.parse(localStorage.getItem("choices"));
+  $("#foodChoice").text(choicesArr[0].food);
+  if (choicesArr[0].event === "") {
```

Comment:
Try to use falsey instead of comparing with an empty string unless you are really interested in the fact that it's a string

```
if (!choices[0].event)
```